src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "PRETTY_NAME" | sed 's/PRETTY_NAME=//g' | sed 's/["]//g' | awk '{print $1}'`
```

Comment:
Sure.. Rahul can take care of all these changes when he sends his PR with the updates required for other distros as well